egs2/librispeech_100/asr_transducer1/run.sh

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env bash
+
+set -e
+set -u
+set -o pipefail
+
+train_set="train_clean_100"
+valid_set="dev"
+test_sets="test_clean test_other dev_clean dev_other"
+
+asr_config=conf/train_conformer-rnn_transducer.yaml
+inference_config=conf/decode.yaml
+inference_model=valid.loss.ave_10best.pth
+
+./asr.sh \
+    --asr_task asr_transducer \
+    --skip_data_prep false \
+    --skip_train false \
+    --skip_eval false \
+    --lang en \
+    --ngpu 1 \
+    --nj 32 \
+    --inference_nj 32 \
+    --nbpe 500 \
```

Comment:
BPE size is `500` vs `5000` for the CTC-Att baseline model. I guess we can further improve results with bigger BPE size but I don't have enough resource for that.